low-level/framework/base/sspl_constants.py

```diff
@@ -74,6 +74,11 @@
 SSPL_CONFIG_INDEX = "SSPL"
 SSPL_TEST_CONFIG_INDEX = "SSPL_TEST"
 CONFIG_SPEC_TYPE = "yaml"
+DEFAULT_DC = "DC01"
+DEFAULT_RACK = "RC01"
+DEFAULT_SN = "SN01"
+DEFAULT_CLUSTER = "CC01"
```

Comment:
When config API is not available to serve input data for sspl, why should we proceed with default value?

Ex. Encrypted password(secret) is based on cluster_id. With default cluster_id enclosure sensor won't get login to the HW enclosure setup as decrypted password would be wrong.